src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -64,6 +64,42 @@ public DarCollectionService(DarCollectionDAO darCollectionDAO, DarCollectionServ
     this.matchDAO = matchDAO;
   }
 
+  /**
+   * Find all DarCollectionSummaries for a given role name.
+   *  Admins can see all summaries
+   *  Chairs and Members can see summaries for datasets they have access to
+   *  Signing Officials can see summaries for researchers in their institution
+   *  Researchers can see only their own summaries
+   *
+   * @param user The user making the request
+   * @param userRole The role the user is making the request as
+   * @return List of DarCollectionSummary objects
+   */
+  public List<Object> getSummariesForRoleName(User user, String userRole) {
+    switch (userRole) {
+      case Resource.ADMIN:
+        // TODO: Find all
+        break;
+      case Resource.SIGNINGOFFICIAL:
+        // TODO: Find by creator institution
+        break;
+      case Resource.CHAIRPERSON:
+      case Resource.MEMBER:
+        List<Integer> dacIds = user.getRoles().stream()
+          .map(UserRole::getDacId)
+          .filter(Objects::nonNull)
+          .collect(Collectors.toList());
+        List<Integer> collectionIds = darCollectionDAO.findDARCollectionIdsByDacIds(dacIds);
```

Comment:
I'm guessing this is a stub as well so I'm leaving a note here. The DAO method for DAC users takes in datasetIds. `DatasetDAO.findDatasetsByUserId` handles the DAC associations for you and will return the datasets that belong to the user's dac.

Reply:
Yes it is just a stub - we can update this in a separate PR.